packages/ui/src/styles/globals.css

```
@@ -8,6 +8,43 @@
+  }
+}
+
+/* Android Chrome, Firefox */
+@media screen and (max-width: 768px) {
+  .h-screen {
+    height: 100dvh; /* dynamic viewport height for modern browsers */
+    height: 100vh;
+  }
+}
+
+/* Safari 및 삼성 브라우저 */
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)